apps/web/src/components/layout/components/SideNav.tsx

```diff
@@ -92,6 +94,7 @@ export function SideNav({}: Props) {
   }, [environment, addItem, removeItems, setEnvironment]);
 
   const handleHideOnboardingClick = async () => {
+    segment.track('Click Hide Get Started Page - [Get Started]');
```

Comment:
it's a good question, the main agenda for doing it like `description - [area]`
is if we add the `area` first we will lose the `description` while searching in our analytic tool. As a user, if I am searching for the `template editor` events I would prefer to see more of the `description` and not the `area` that I am searching for.
![image](https://github.com/novuhq/novu/assets/39195835/7a7646f9-0052-4545-b2e0-db818b4833e1)

wdyt?

Reply:
Hmm I'm a little torn on this since I think it's important that we stay consistent one way or another. Also, perhaps there is a better way to track these prefixes /tags

Reply:
It should be in the format `xxx - [yyy]`, that was the decision we took some time ago. All other places were added before and we didn't want to "lose" some data because of that change.

Reply:
This smells of something that should have been refactored when when that decision was made 😅

Ticket here, please add your thoughts: https://linear.app/novu/issue/NV-3352/%F0%9F%99%8B-analytics